### PR TITLE
Removed TODO from test-case for hard-coded delivery fee estimation

### PR DESCRIPTION
**Pull request by bkontur**

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4093586

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4093843

**Review by acatangiu — APPROVED** [review]

File: cumulus/parachains/runtimes/assets/asset-hub-rococo/tests/tests.rs [hunk 1]
```diff
@@ -683,6 +683,7 @@ mod asset_hub_rococo_tests {
 			bridging_to_asset_hub_wococo,
 			WeightLimit::Unlimited,
 			Some(xcm_config::bridging::XcmBridgeHubRouterFeeAssetId::get()),
+			Some(xcm_config::TreasuryAccount::get().unwrap()),
```
Comment (serban300) [inline cumulus/parachains/runtimes/assets/asset-hub-rococo/tests/tests.rs:686]: TreasuryAccount is an `Option` right now. I think we could simply use `xcm_config::TreasuryAccount::get()`

**Review by serban300 — APPROVED** [review]

File: cumulus/parachains/runtimes/assets/asset-hub-rococo/tests/tests.rs (hunk 1 above)
Comment (bkontur) [reply cumulus/parachains/runtimes/assets/asset-hub-rococo/tests/tests.rs:686]: yes, I know, but I wanted to be explicit that this test uses `Some`

**Pull request merged**